test/unit/test_devices/test_ios_device.py

```diff
@@ -348,12 +348,121 @@ def test_enable_from_config(self):
     unittest.main()
 
 
-def test_send_command_timing(ios_send_command_timing):
-    command = "send_command_timing"
+@mock.patch.object(IOSDevice, "redundancy_state", new_callable=mock.PropertyMock)
+@pytest.mark.parametrize(
+    "redundancy_state,expected",
+    (
+        ("active", True),
+        ("standby hot", False),
+        (None, True),
+    ),
+    ids=("active", "standby_hot", "unsupported"),
+)
+def test_is_active(mock_redundancy_state, redundancy_state, expected, ios_device):
```

Comment:
Should there be a `test_is_not_active` for the `False` return or would this be too trivial?

Reply:
@mundruid that is covered by the second test case in the parametrize decorator.